monai/handlers/checkpoint_saver.py

```diff
@@ -62,6 +62,9 @@ class CheckpointSaver:
             typically, it's used to resume training and compare current metric with previous N values.
         key_metric_greater_or_equal: if `True`, the latest equally scored model is stored. Otherwise,
             save the the first equally scored model. default to `False`.
+        key_metric_score_sign: sign of the score, available value: `[1.0, -1.0, 1, -1]`. For error-like metrics,
```

Comment:
I think this should be a flag indicating whether to flip the sign of the metric value, if it is true or positive, the sign of the metric value won't be flipped, if it's false or negative, the metric value will be flipped. It's not about supporting 1.0 -1.0 1 -1 here.

Reply:
Hi @wyli ,

Thanks for your explanation. I tried to align with ignite API: https://github.com/pytorch/ignite/blob/master/ignite/handlers/checkpoint.py#L585
I am OK with both solutions, your idea seems more robust, no need to check the arg value.

Thanks.

Reply:
Let me tried to update it to bool flag in this ignite related PR together: https://github.com/Project-MONAI/MONAI/pull/2492

Thanks.

Reply:
That docstring seems to suggest that it'll ensure a positive or negative value, but the actual implemention is about 'flip or not'. Perhaps the ignite docstring needs an update as well...

Reply:
Hi @vfdev-5 , what do you think about this arg? Maybe ignite can also change it to be a bool flag?

Thanks.

Reply:
@wyli , updated in the latest commit.
Thanks.

Reply:
> what do you think about this arg? Maybe ignite can also change it to be a bool flag?

Hi @Nic-Ma , this is a good question. Let me see with the team about that.